frontend/src/components/App/App.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import { Route } from 'react-router-dom';
-import SignInPage from './SignInPage/SignInPage';
-import SignUpPage from './SignUpPage/SignUpPage';
+import { SignInPage } from '../SignInPage';
```

Comment:
Update component import paths in `App.js` to reflect added `src/component/index.js`.